mne/preprocessing/tests/test_maxwell.py

```diff
@@ -1443,3 +1444,46 @@ def test_compute_maxwell_basis(regularize, n):
     xform = S[:, :n_use_in] @ pS[:n_use_in]
     got = xform @ raw.pick_types(meg=True, exclude='bads').get_data()
     assert_allclose(got, want)
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize(
+    'bads', ['from_raw', 'union', ['MEG 0113', 'MEG 2313']]
+)
+def test_prepare_emptyroom(bads):
+    """Test prepare_emptyroom."""
+    raw = read_raw_fif(sample_fname)
+    raw_er = raw.copy().pick_types(meg=True)
+    raw_er.info['dev_head_t'] = None
+    raw_er.set_montage(None)
+
+    if bads == 'from_raw':
+        raw_bads_orig = ['MEG 0113', 'MEG 2313']
+        raw_er_bads_orig = []
+    elif bads == 'union':
+        raw_bads_orig = ['MEG 0113']
+        raw_er_bads_orig = ['MEG 2313']
+    else:
+        raw_bads_orig = bads
+        raw_er_bads_orig = []
+
+    raw.info['bads'] = raw_bads_orig
+    raw_er.info['bads'] = raw_er_bads_orig
+
+    raw_er_prepared = maxwell_filter_prepare_emptyroom(
+        raw_er=raw_er,
+        raw=raw,
+        bads=bads
+    )
+    assert raw_er_prepared.info['bads'] == ['MEG 0113', 'MEG 2313']
+    assert raw_er_prepared.info['dev_head_t'] == raw.info['dev_head_t']
+
+    montage_expected = raw.copy().pick_types(meg=True).get_montage()
+    # XXX assert raw_er.get_montage == montage_expected fails – why?
```

Comment:
@agramfort any idea on this one?

Reply:
It should be the prepared one no ?

Reply:
Yes, I forgot to update the commented-out line after renaming the variable.

But even the following fails:
```python
    assert raw_er_prepared.get_montage() == montage_expected
```
directly comparing the `.dig` attributes, OTOH, works.
So I'm not sure what's going on here

Reply:
You can dig into what `__eq__` does for the class to figure it out

Reply:
agreed. Just use an == with dig key in info

Reply:
> You can dig into what `__eq__` does for the class to figure it out

Hah. Turns out `DigMontage` inherits directly from `object` and doesn't implement its own `__eq__()`. We should probably put this on our todo.

In any case, I'll just compare `.dig` for now, which should suffice.